vision/transforms/to_tensor.go

```diff
@@ -58,7 +58,7 @@ func colorImageToTensor(img image.Image) torch.Tensor {
 		}
 	}
 	hwc := torch.FromBlob(unsafe.Pointer(&array[0]), torch.Float,
-		[]int64{int64(maxY), int64(maxX), 3})
+		[]int64{int64(maxY), int64(maxX), 3}).Clone()
```

Comment:
We have to call `Clone` here, because we create a temporary Go slice `array := make([]float32, maxY*maxX*3)` here. 

If we use the data in `p.images` directly, `Clone` will be removed safely.